scripts/damage-log.js

```diff
@@ -144,6 +144,25 @@ class DamageLog {
 				min: "health.min",
 				max: "health.max"
 			}
+		},
+		ac2d20: {
+			stress: {
+				invert: true,
+				value: "stress.value",
+				max: "stress.max"
+			},
+			injuries: {
+				invert: true,
+				value: "injuries.value"
+			},
+			fortune: {
+				invert: false,
+				value: "fortune.value"
+			},
+			fatigue: {
+				invert: true,
+				value: "fatigue"
+			}
```

Comment:
Can you move this so the list is in alphabetical order please?

Reply:
Fixed: https://github.com/cs96and/FoundryVTT-damage-log/pull/42/commits/3922d88427048bcf4e1794a1891c895a724f4820